frontend/app/routes/$lang+/_protected+/alerts+/subscribe+/confirm.tsx

```diff
@@ -184,8 +184,7 @@ export default function ConfirmSubscription() {
                 </div>
               </>
             ) : (
-              //TODO add content: https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/3349
-              'Requested'
+              <p>{t('alerts:confirm.code-sent-by-email', { userEmailAddress: alertSubscription?.email })}</p>
```

Comment:
The optional chaining suggests that the email can be null/undefined here. Since the rendered text won't make sense if it is, I think you should wrap the entire block in a conditional and only render when `email` is defined.